manifest.json

```diff
@@ -1,5 +1,809 @@
+                                "type": "string"
+                              },
+                              {
+                                "const": "2_CONTRIBUTOR",
+                                "type": "string"
+                              },
+                              {
+                                "const": "2_SPECIFICATION",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_4",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_8",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_16",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_32",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_128",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_ASSIGNEE",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_AUTHOR",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_COLLABORATOR",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_CONTRIBUTOR",
+                                "type": "string"
+                              },
+                              {
+                                "const": "PULL_SPECIFICATION",
+                                "type": "string"
+                              },
+                              {
+                                "const": "ISSUE_4",
+                                "type": "string"
+                              },
+                              {
+                                "const": "ISSUE_8",
+                                "type": "string"
+                              },
+                              {
+                                "const": "ISSUE_16",
+                                "type": "string"
+                              },
+                              {
+                                "const": "ISSUE_32",
+                                "type": "string"
+                              },
+                              {
+                                "const": "ISSUE_128",
```

Comment:
These look like they are from the tests. This doesn't look right.

Reply:
No it's not from the test. It's because these are dynamically generated types that JSON recognizes as values. Technically `ISSUE_128` and `ISSUE_ASSIGNEE` would be the same. These are the limitations we have from conveying the information through JSON. To handle this we should write custom transformers.